lib/nosedrum/application_command.ex

```diff
@@ -269,6 +269,18 @@ defmodule Nosedrum.ApplicationCommand do
   """
   @callback options() :: [option]
 
+  @doc """
+    An optional callback that returns a bitset for the required default permissions to run this command.
+
+    Example callback that requires that the user has the permission to ban members to be able to see and execute this command
+
+    ```elixir
+    def default_member_permissions, do:
+      Nostrum.Permission.to_bitset([:ban_members])
+    ```
+  """
+  @callback default_member_permissions() :: integer
```

Comment:
The docs mention that this should be a string: https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-structure
Should we cast it in the `function_exported?` block below? Or did you test it this way and it works just fin
e?